src/containers/PostPageVolunteer/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import PostPageVolunteerHeader from "../../components/PostPageVolunteerHeader";
+import PostList from "../../components/PostList";
+
+export default function PostPageVolunteer() {
+  return (
+    <div>
+      <PostPageVolunteerHeader />
+      <PostList collectionName="donatorPost" />
```

Comment:
What about the receiver post, where are you passing the props to them?

Reply:
It is another container <PostPageReciever>.In that container the same way, I am passing the props as "recieverPost".

Reply:
Cool, I see the. Wait until this merged then pull for the other request, instead of add the whole code again